vite.config.ts

```
@@ -14,6 +14,13 @@ export default defineConfig({
   ],
   server: {
     port: 3000,
+    proxy: {
+      '/api': {
```

ESLint

[error] Replace `'/api'` with `"/api"`

(prettier/prettier)